experiments/standalone/Bucket/global_bucket_function.jl

```diff
@@ -164,6 +169,38 @@ updatefunc = ClimaLand.make_update_drivers(bucket_atmos, bucket_rad)
 driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
 cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
 
+
+
+# ClimaDiagnostics
+base_output_dir = "global_bucket_function/"
+output_dir =
+    ClimaUtilities.OutputPathGenerator.generate_output_path(base_output_dir)
+
+space = bucket_domain.space.subsurface
+
+nc_writer = ClimaDiagnostics.Writers.NetCDFWriter(space, output_dir)
+
+diags = ClimaLand.CLD.default_diagnostics(model, 1.0; output_writer = nc_writer)
+
+diagnostic_handler =
+    ClimaDiagnostics.DiagnosticsHandler(diags, Y, p, t0; dt = Δt)
+
+diag_cb = ClimaDiagnostics.DiagnosticsCallback(diagnostic_handler)
+
+sol_test = SciMLBase.solve(prob, ode_algo; dt = Δt, callback = diag_cb)
+
+#### ClimaAnalysis ####
+
+simdir = ClimaAnalysis.SimDir(output_dir)
+println(summary(simdir))
+alpha = get(simdir; short_name = "alpha")
+alpha.dims
+fig = CairoMakie.Figure(size = (400, 600))
+viz.plot!(fig, alpha, time = 2 * 86400)
+CairoMakie.save(joinpath(output_dir, "alpha.png"), fig)
```

Comment:
```suggestion
for short_name in ["alpha", "W"]
var = get(simdir; short_name)
fig = CairoMakie.Figure(size = (400, 600))
viz.plot!(fig, alpha, time = 2 * 86400)
CairoMakie.save(joinpath(output_dir, "$short_name.png"), fig)
end
```

Reply:
Done and it works, but two issues: 

- tsoil is 3D (has depth) so viz.plot! requires z. But other variables don't have z. For now I just commented out tsoil, but this is an easy fix.
- more importantly, in this experiment it seems all variables are constant in space, so the heatmap just shows one color.